### ISSUE TITLE
Improve complex Array diffs

### ISSUE THREAD

**Issue by lukeed** [lukeed/uvu#1]
Come back to this --> have `diff.js` locally with few different approaches. Decide on large-scale changes. Keep whitespace character normalization

**Comment by jasikpark** [comment]
https://github.com/banga/git-split-diffs is there a requirement for minimal deps for this project? I found this project in the "related projects" section of `delta`, and it seems solid, though I don't see a recommended way to use it programmatically.

**Comment by lukeed** [comment]
@jasikpark Yes, definitely. That package is using the same `diff` dependency under the hood anyway. The only difference is the visual output, which is side-by-side (by default) and using the background highlights that you like :)

**Comment by jasikpark** [comment]
I'm probably just biased from Github diffs 😂😅

**Comment by jasikpark** [comment]
Perhaps just yoink <https://avajs.dev>'s diff package? https://github.com/concordancejs/concordance

**Comment by lukeed** [comment]
I am not interested in external dependencies or solutions. 85% of the fun is coming up with something.
